src/main/java/mate/academy/spring/model/User.java

```diff
@@ -16,6 +18,8 @@ public class User {
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany()
+    private Set<Role> roles;
```

Comment:
It is better to set table and column names manually using annotation @Table ...
Autogenerated namings could benamed not according to standarts.